<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ameet" failures="11" tests="11" name="Test" time="61.018" errors="0" timestamp="2025-07-11T13:08:42 IST">
  <testcase classname="testcases.RestAssured_TestCases" name="loginWithSeleniumAndGetCookie" time="33.463">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;css selector&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;*[name=&amp;apos;username&amp;apos;]&amp;quot;}
  (Session info: chrome=138.0.7204.97)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &amp;apos;4.10.0&amp;apos;, revision: &amp;apos;c14d967899&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.7&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [821c145c6654ac0cce20d5b4205579fc, findElement {using=name, value=username}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.97, chrome: {chromedriverVersion: 138.0.7204.94 (545025748884..., userDataDir: C:\Users\AMEETK~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51023}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51023/devtoo..., se:cdpVersion: 138.0.7204.97, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 821c145c6654ac0cce20d5b4205579fc">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"*[name='username']"}
  (Session info: chrome=138.0.7204.97)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [821c145c6654ac0cce20d5b4205579fc, findElement {using=name, value=username}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 138.0.7204.97, chrome: {chromedriverVersion: 138.0.7204.94 (545025748884..., userDataDir: C:\Users\AMEETK~1\AppData\L...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51023}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51023/devtoo..., se:cdpVersion: 138.0.7204.97, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 821c145c6654ac0cce20d5b4205579fc
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:199)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:132)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:51)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:531)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:66)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at testcases.RestAssured_TestCases.loginWithSeleniumAndGetCookie(RestAssured_TestCases.java:67)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- loginWithSeleniumAndGetCookie -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetHolidayData" time="0.164">
    <failure type="java.lang.AssertionError" message="GetHolidayData must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: GetHolidayData must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.GetHolidayData(RestAssured_TestCases.java:103)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- GetHolidayData -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetLeaveData" time="0.01">
    <failure type="java.lang.AssertionError" message="GetLeaveData must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: GetLeaveData must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.GetLeaveData(RestAssured_TestCases.java:154)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- GetLeaveData -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetEmpCount" time="0.008">
    <failure type="java.lang.AssertionError" message="GetEmpCount must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: GetEmpCount must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.GetEmpCount(RestAssured_TestCases.java:200)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- GetEmpCount -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetLeaveType" time="0.019">
    <failure type="java.lang.AssertionError" message="GetLeaveType must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: GetLeaveType must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.GetLeaveType(RestAssured_TestCases.java:237)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- GetLeaveType -->
  <testcase classname="testcases.RestAssured_TestCases" name="GetUsageReport" time="0.019">
    <failure type="java.lang.AssertionError" message="GetUsageReport must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: GetUsageReport must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.GetUsageReport(RestAssured_TestCases.java:284)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- GetUsageReport -->
  <testcase classname="testcases.RestAssured_TestCases" name="PutTerminationReason" time="1.877">
    <failure type="java.lang.AssertionError" message="PutTerminationReason must be implemented using RestAssured methods only! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: PutTerminationReason must be implemented using RestAssured methods only! expected [true] but found [false]
at testcases.RestAssured_TestCases.PutTerminationReason(RestAssured_TestCases.java:343)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- PutTerminationReason -->
  <testcase classname="testcases.RestAssured_TestCases" name="DeletePim" time="9.013">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)" is null
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at testcases.RestAssured_TestCases.DeletePim(RestAssured_TestCases.java:375)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- DeletePim -->
  <testcase classname="testcases.RestAssured_TestCases" name="PostPimEmp" time="0.411">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;io.restassured.response.Response.jsonPath()&amp;quot; because the return value of &amp;quot;rest.CustomResponse.getResponse()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.jsonPath()" because the return value of "rest.CustomResponse.getResponse()" is null
at testcases.TestCodeValidator.validateResponseFields(TestCodeValidator.java:301)
at testcases.RestAssured_TestCases.PostPimEmp(RestAssured_TestCases.java:429)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PostPimEmp -->
  <testcase classname="testcases.RestAssured_TestCases" name="PutVimEmp" time="1.122">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)" is null
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at testcases.RestAssured_TestCases.PutVimEmp(RestAssured_TestCases.java:456)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- PutVimEmp -->
  <testcase classname="testcases.RestAssured_TestCases" name="DeletePimEmp" time="3.254">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "io.restassured.internal.common.path.ObjectConverter.convertObjectTo(Object, java.lang.Class)" is null
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at testcases.RestAssured_TestCases.DeletePimEmp(RestAssured_TestCases.java:507)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- DeletePimEmp -->
</testsuite> <!-- Test -->
